packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml

```diff
@@ -15,12 +15,20 @@ spec:
       labels:
         name: {{ .Release.Name }}-rke2-journald-aggregator
     spec:
+      {{- if .Values.global.seLinux.enabled }}
+      securityContext:
+        seLinuxOptions:
+          type: rke_logreader_t
+      {{- end }}
```

Comment:
nit: this is fine since it's just one container but to be consistent with the rke aggregator it would be nice to put it at the container level rather than the pod level https://github.com/rancher/charts/blob/2420e452723d064580ff839105b25131c5954a86/charts/rancher-logging/rancher-logging/3.9.002-rc01/templates/loggings/rke/daemonset.yaml#L32-L36

Reply:
I will make the change now, it will avoid a misunderstanding in the future.